public/app/features/explore/state/query.ts

```diff
@@ -904,6 +912,28 @@ export const queryReducer = (state: ExploreItemState, action: AnyAction): Explor
   return state;
 };
 
+/**
+ * Creates an observable that emits correlations once they are loaded
+ */
+const ensureCorrelations = () => {
```

Comment:
nit: what about
```suggestion
const getCorrelations$ = () => {
```
?

seems to be more inline with rxjs naming conventions

Reply:
Fixed in ddaac692a3de1fe3dcfbebb4a6c4dafadcb757bf